specs/PointOfInterest/PointOfInterest/schema.json

```diff
@@ -5,6 +5,15 @@
   "description": "A point of interest",
   "type": "object",
   "allOf": [
+    {"anyOf": [{"required": ["location"]}, {"required": ["address"]}]},
```

Comment:
please indent properly

you can use https://www.jsonschemavalidator.net/

Reply:
Fixed in aca43da